frontend/src/components/UserDetails.vue

```diff
@@ -93,20 +93,6 @@
               {{ identity.identityProvider | formatIdentityProvider }} [{{ identity.userName }}]
             </li>
           </ul>
-          <br/><br/>
-          <label for="all-roles">Application Roles and Last Login</label>
-          <v-skeleton-loader
-              ref="roleSkeleton"
-              v-show="!rolesLoaded"
-              type="list-item@5">
-          </v-skeleton-loader>
-          <div id="user-roles" v-show="rolesLoaded">          
-            <ul id="all-roles" style="margin-top: 5px; list-style: square">
-              <li v-for="client in allRoles" :key="client.clientName">
```

Comment:
The method (`loadUserRoles`) that populates `allRoles` can be removed, as well as the data variables for `allRoles` and  `rolesLoaded`